src/_actions/WorkspaceActions.js

```diff
@@ -55,3 +55,11 @@ export const delFromWatchlist = symbol => {
         dispatch(workspaceUnfavorAsset(symbol));
     };
 };
+
+export const selectAssetSymbolForTrade = (newSymbol, oldSymbol) => {
+    return dispatch => {
+        LiveData.api.unsubscribeFromTick(oldSymbol);
+        LiveData.api.subscribeToTick(newSymbol);
```

Comment:
well, I follow recommendation from the creator

https://github.com/rackt/redux/issues/291

any reason not to follow? or do you mean we should abstract it further?